ranger-client/src/main/java/io/appform/ranger/client/AbstractRangerHubClient.java

```diff
@@ -48,6 +49,9 @@ public abstract class AbstractRangerHubClient<T, R extends ServiceRegistry<T>, D
     private ServiceDataSource serviceDataSource;
     private long serviceRefreshDurationMs;
     private long hubRefreshDurationMs;
+    private long serviceRefreshTimeoutMs;
+    private long hubStartTimeoutMs;
```

Comment:
Let's also add a bit document  about hubStartTimeoutMs, serviceRefreshTimeoutMs and hubRefreshDurationMs.

Reply:
Added comments for description about the parameters